lib/swifft_object.js

```diff
@@ -53,6 +62,24 @@ export default class SwiftObject {
         });
     }
 
+    getRange(range, callback) {
+
+        let opts = {
+            method: 'GET',
+            //payload: content,
+            headers: ObjectSettings.toHeaders(range)
```

Comment:
Even though this argument is named `range` it appears that it expects an [explicitly structured object](https://github.com/totherik/swifft/pull/2/files#diff-c4912838eca48e56a1000a78da9532aaR69) with `range` as a property (thus the argument itself isn't "range").

Perhaps, this method should accept a string value for `range` and wrap it appropriately:

```js
let opts = {
    method: 'GET',
    headers: ObjectSettings.toHeaders({ range })
};

```